cookbooks/hls_nginx/attributes/default.rb

```diff
@@ -4,5 +4,7 @@
   "server_root" => "/var/www/html/public",
   "error_location" => "/var/www/html/public",
   "puma_upstream" => "unix:///tmp/puma.sock",
-  "dummy_upstream" => "127.0.0.1:4567"
+  "dummy_upstream" => "127.0.0.1:4567",
+  "websocket_support" => "false",
```

Comment:
I think we can make this an actual boolean:

```suggestion
  "websocket_support" => false,
```